apps/dep-graph-client/src/app/graph.ts

```diff
@@ -0,0 +1,384 @@
+import { default as tippy, Instance } from 'tippy.js';
```

Comment:
This file is very long :eyes: 

Below are ideas/ suggestions how we can tame this huge file and make it cleaner/ more maintainable. Feel free to implement them differently.